### PR TITLE
fix(eslint): Improve vue arch creation and solve eslint

### PR DESCRIPTION
**Pull request by alexbruno**
Changes to improve Vue arch building and solve eslint errors into a fresh project with gandalf-lint.

### CODE REVIEW

File: src/templates/vue/js-def-cfgs.ts [hunk 1]
```diff
@@ -67,3 +73,18 @@ function eslint(config: any) {
 
   return config;
 }
+
+// Workarounds
```
[the remark this answers is not in the document]
Comment (alexbruno) [reply src/templates/vue/js-def-cfgs.ts]: This small pat of code was not passing gandalf-lint

File: src/templates/vue/js-def-cfgs.ts [hunk 2]
```diff
@@ -9,26 +9,32 @@ export default function jsDefCfgs(dir: string) {
...
-    'We started a new MOOvelous project!',
-    style.Bright,
-    style.Blue,
-  );
+  writeFileSync(pack, result, 'utf8');
 }
 
-// Helpers
-function eslint(config: any) {
+function eslintSettings(config: any) {
```
[the remark this answers is not in the document]
Comment (alexbruno) [reply src/templates/vue/js-def-cfgs.ts]: @vilaboim and me were experienced problematic rules with default gandalf-lint into a @vue/cli project. These settings are "solutions" that works for us.

File: src/templates/vue/js-def-deps.ts [hunk 3]
```diff
@@ -14,10 +14,10 @@ export default function jsDefDeps(dir: string) {
   const task = spawn(
     'npm i -D',
     [
-      '@softboxlab/eslint-config-gandalf-lint',
```
[the remark this answers is not in the document]
Comment (alexbruno) [reply src/templates/vue/js-def-deps.ts]: Ok, let's try it.

File: src/templates/vue/js-def-cfgs.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (pablohpsilva) [reply src/templates/vue/js-def-cfgs.ts]: I got you, but... I believe you can find the same thing on the recently publish package `gandalf-lint-vue`. @vilaboim did it himself I guess.

**Pull request merged**